src/aggregate/aggregate_plan.c

```diff
@@ -172,12 +195,166 @@ void AGPLN_Dump(const AGGPlan *pln) {
   for (const DLLIST_node *nn = pln->steps.next; nn && nn != &pln->steps; nn = nn->next) {
```

Comment:
Do you think making a command from it and return it as a result instead of printing it?
Not saying we should do it now but maybe in the future ...

Reply:
Kind of like `EXPLAIN`? yeah, that would be a good idea in the future.